config/areas/site.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+return function ($kirby) {
+    return [
+        'breadcrumbLabel' => function () use ($kirby) {
+            return $kirby->site()->title()->or(t('view.site'))->toString();
+        },
+        'icon'   => 'home',
+        'label'  => t('view.site'),
+        'menu'   => true,
+        'routes' => [
+            [
+                'pattern' => 'pages/(:any)',
+                'action'  => function (string $path) use ($kirby) {
+                    if (!$page = $kirby->page(str_replace('+', '/', $path))) {
```

Comment:
```suggestion
                    $page = $kirby->page(str_replace('+', '/', $path));

                    if (!$page) {
```

Reply:
If we rewrite these, we should actually make the if statement consistent with our code styles, e.g. explicitly check for `$page === null` in this case then.

Reply:
Being less strict makes sense here – any falsy value should lead to an error, not just `null`. But I‘m fine with either solution.